exercises/06-using-service-account/solution/solution.md

```diff
@@ -22,7 +22,7 @@ secrets:
 Next, you can create a new Pod and assign the service account to it.
 
 ```shell
-$ kubectl run backend --image=nginx --restart=Never --serviceaccount=backend-team
+$ kubectl run backend --image=nginx --restart=Never --overrides='{ "spec": { "serviceAccount": "backend-team" } }'
```

Comment:
The attribute `serviceAccount` in the PodSpec is [deprecated](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#pod-v1-core).

> ServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.

Can you please change this to `serviceAccountName`?

Reply:
Corrected to `serviceAccountName`. Thanks!